src/modules/assignment-4/questions/question8/Task8.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import "../Styles.css";
+
+const Task8 = () => {
+  const [tempC, setTempC] = useState(0);
+  const [tempF, setTempF] = useState(0);
+
+  const handleUserTemp = (e) => {
+    const { id, value } = e.target;
+
+    if (id === "celsius") {
+      setTempC(value);
+      setTempF((value * 9) / 5 + 32);
+    } else {
+      setTempC(((value - 32) * 5) / 9);
+      setTempF(value);
+    }
+  };
+
+  return (
+    <>
+      <h2>TEMPERATURE CONVERTER</h2>
+
+      <input
+        type="text"
+        value={tempC}
+        id="celsius"
+        placeholder="Celsius"
+        onChange={handleUserTemp}
+      />
+
+      <input
+        type="text"
+        value={tempF}
```

Comment:
![image](https://github.com/ayush1-successive/react-training/assets/83696224/273b5400-deff-46b3-bacd-ec96f1e2b8fc)
Mention on the input fields which one is Celsius and which one is Fahrenheit